recipes/htmllistparse/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Can you open a PR upstream to include this in the `MANIFEST.in` so it's in future tarballs?

(Technically it's a requirement of the licence and generally it's better to have it there in case the licence changes)

Reply:
https://github.com/gumblex/htmllisting-parser/pull/6

Reply:
Thanks! 🍰